src/sider/historikk/container/HistorikkContainer.tsx

```diff
@@ -144,6 +147,12 @@ function manglendemedvirkningText(
   }
 }
 
+function friskTilArbeidText(friskTilArbeid: VedtakResponseDTO): string {
```

Comment:
Hadde det vært mer rett frem med `vedtak`? 😊
```suggestion
function friskTilArbeidText(vedtak: VedtakResponseDTO): string {
```